package.json

```diff
@@ -26,7 +26,8 @@
     ".": {
       "require": "./dist/index.js",
       "import": "./dist/react-use-intercom.esm.js",
-      "browser": "./dist/react-use-intercom.esm.js"
+      "browser": "./dist/react-use-intercom.esm.js",
+      "types": "./dist/index.d.ts"
```

Comment:
Can you move this to the first line, according to https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing the `types` entry must occur first

Reply:
Of course! It has been updated now

Reply:
Thanks